miwa.rb

```diff
@@ -0,0 +1,45 @@
+class Indicatory
+    puts 'Щосаме вам потрібно порахувати ?'
+    answer = gets.chomp.upcase 
+    if answer == 'Electric'
+      @electric = gets.chomp 
+    elsif answer == 'Gas'
+      @gas = gets.chomp
+    elsif answer = 'Water'
+      @coldWater = gets.chomp
+      @hotWater = gets.chomp
+    else 
+        puts 'Виберіть одну з '
+    end 
+end
+
+class Electric
```

Comment:
We need to create a basic class, `Product` for example, it should have all the common methods. `calculate` etc. And other classes should inherit it.